### DISCUSSION TITLE
Bug uppercase extension

### DISCUSSION THREAD

**Discussion by Topunzel** [ImageMagick/ImageMagick#5560] [General] [answered]
Hello, 

i think this is the wrong place for a bug, but i can't register on the forum, sooo....

If a file has the extension jpg but wrote in uppercase JPG, imagick does not write the exif data of the original file. You need to manually convert the extension to lowercase.

Best regards,
Tobias

**Comment by snibgo** [comment, marked as answer]
> ... imagick does not write the exif data ...

If the problem is with IMagick, then this is the wrong place to report the bug.

This is an ImageMagick forum. IMagick is a different product, an interface between PHP and ImageMagick.

If the problem can be reproduced with just ImageMagick, then please report it at https://github.com/ImageMagick/ImageMagick/issues . Give enough information that the problem can be reproduced.
